packages/dcos-diagnostics/windows.buildinfo.json

```diff
@@ -0,0 +1,8 @@
+{
+  "single_source" : {
+    "kind": "git",
+    "git": "https://github.com/dcos/dcos-diagnostics.git",
+    "ref": "1b3be79e72185db0c9786d0d994984eb5d1efc4b",
```

Comment:
This line duplicates https://github.com/dcos/dcos/blob/e2e77eebe36405e60e3eb1c309d0b86925bef7af/packages/dcos-diagnostics/buildinfo.json#L6 and makes it harder to bump dcos-diagnostics in the future. We should find a way to unify it.

Reply:
I don't think that is possible. Please look [here](https://github.com/dcos/dcos/tree/master/packages/mesos), mesos has as well both files.